libs/components/tabs/src/lib/modules/tabs/tab-button.component.scss

```diff
@@ -11,6 +11,7 @@
     top: 10px;
     right: 10px;
     color: $sky-text-color-deemphasized;
+    padding: 1px 6px;
```

Comment:
We were inheriting the padding that the browser gives native `button` elements. Adding it here since we're only using a span now.